### ISSUE TITLE
change loss object detection api

### ISSUE THREAD

**Issue by zeynali** [tensorflow/models#3395]
Hi.
i want to change loss of object detection for ones of object detection (such as SSD) ,
Q1 :  i want to know where do i modify the loss function for SSD , 
Q2 : is it possible to fine-tune ssd_mobilenet on my dataset with my define loss ? is it good or must be train ssd_mobile from scratch with my loss function ? 

in the ssd_mobilenet_v1_coc.config : 
    loss {
      classification_loss {
        weighted_sigmoid {
          anchorwise_output: true
        }
      }
i want to change classification loss to Focal loss , what do i do ?

**Comment by gsssrao** [comment]
It is already present in [object_detection/samples/configs/ssd_mobilenet_v1_focal_loss_coco.config](https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/ssd_mobilenet_v1_focal_loss_coco.config)

**Comment by zychen2016** [comment]
if a New loss , what do i do?

**Comment by zychen2016** [comment]
if a New loss , what do i do?
